magento/magento2
# Fixed rest API products stockitem /V1/products?searchCriteria  #22737

### Description (*)


Magento 2.3 This pull request (PR) will be merge the added stock item extension_attributes in product list rest API /rest/V1/products?searchCriteria

### Fixed Issues (if relevant)

1. Fixes #22737,  : "/rest/V1/products?searchCriteria" API endpoint" does not return information about stock in the "extension_attributes" section
2. Fixes #24418

### Manual testing scenarios (*)

1. Open the Postman
2. Add the API URL: {baseurl}/rest/V1/products?searchCriteria[filterGroups][0][filters][0][field]=category_id& searchCriteria[filterGroups][0][filters][0][value]=3& searchCriteria[filterGroups][0][filters][0][conditionType]=eq&searchCriteria[sortOrders][0][field]=price& searchCriteria[sortOrders][0][direction]=ASC

3. Method: GET
4. Authorization "Use Admin Token"
5. Check in response There is a "stock_item": object added in extension_attributes section.

### Questions or comments


### Contribution checklist (*)
